locales/en-US.json

```
@@ -2,6 +2,8 @@
   "alphabetically": "Alphabetically",
   "badRequest": "Bad request. Try again",
   "browserNotSupported": "Your browser is not supported for playing this file.",
+  "changeServer": "Change Server",
```

```suggestion
  "changeServer": "Change server",
```

We're not doing title casing anywhere else :yum:

---

🤦‍♂️ I do this wrong too often 😝